app/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/julianjca/julian-golang-training-beginner/app/cmd"
+
+func main() {
+	cmd.Execute()
```

Comment:
this pattern is spf13/cobra specific pattern, no need to use it here if not using cobra